src/base/bittorrent/trackerentry.cpp

```diff
@@ -52,6 +53,30 @@ QString TrackerEntry::url() const
     return QString::fromStdString(nativeEntry().url);
 }
 
+QStringList TrackerEntry::messages() const
+{
+    QStringList messages;
```

Comment:
Is the order of the message important? If not, you might want to use `QSet<QString>` as I think `QStringList::contains()` could be expensive.

Reply:
I agree set seems natural, but this list will have at most few entries, so actually set is really not bringing anything to table, it it will likely be slower in fact. Hash tables are great, but not to compare few elements, I did some quick measurements and it is actually slower. And one other thing is that we would need to convert to list before join anyway, so that would be awkward.

Reply:
How about adding a short comment after the variable to remind future devs that it will contain only a few entries? As to avoid unnecessary changes in the future.

Reply:
Done